### PR TITLE
UICHKIN-288: Add Jest/RTL tests for `ModalManager` component

### PR DESCRIPTION
**Pull request by Stanislau-Snapok**
## Purpose
Add Jest/RTL testing for `ModalManager` component.

## Refs
https://issues.folio.org/browse/UICHKIN-288

## Screenshots
image

### CODE REVIEW

File: src/ModalManager.test.js [hunk 1]
```diff
@@ -0,0 +1,495 @@
...
+
+        it('should close "ClaimedReturnedModal" when onConfirm callback is triggered', () => {
+          const modal = screen.getByTestId(testIds.claimedReturnedModal);
+
+          fireEvent.click(getById(testIds.claimedReturnedModal).getByText(labelIds.confirmButton));
+
+          expect(modal).not.toBeInTheDocument();
+        });
+
+        describe('after "ClaimedReturnedModal" confrimation', () => {
```
Comment (artem-blazhko) [inline src/ModalManager.test.js:247]:
```suggestion
        describe('after "ClaimedReturnedModal" confirmation', () => {
```

**Review by artem-blazhko — APPROVED** [review]

File: src/ModalManager.test.js [hunk 2]
```diff
@@ -0,0 +1,495 @@
...
+      render(
+        <ModalManager
+          {...defaultProps}
+          checkinNotesMode
+        />
+      );
+    });
+
+    afterEach(() => {
+      jest.clearAllMocks();
```
Comment (pavelspichonak) [inline src/ModalManager.test.js:132]: looks like it should be in the first describe?

File: src/ModalManager.test.js [hunk 3]
```diff
@@ -0,0 +1,495 @@
...
+          expect(modal).not.toBeInTheDocument();
+        });
+
+        describe('after "ClaimedReturnedModal" confrimation', () => {
+          beforeEach(() => {
+            fireEvent.click(getById(testIds.claimedReturnedModal).getByText(labelIds.confirmButton));
+          });
+
+          afterEach(() => {
+            jest.clearAllMocks();
```
Comment (pavelspichonak) [inline src/ModalManager.test.js:253]: we use it twice, I would move it to the first describe

**Review by pavelspichonak — APPROVED** [review]

File: src/ModalManager.test.js (hunk 1 above)
Comment (Stanislau-Snapok) [reply src/ModalManager.test.js]: Done, thanks!

File: src/ModalManager.test.js (hunk 2 above)
Comment (Stanislau-Snapok) [reply src/ModalManager.test.js]: You are right, thanks!

File: src/ModalManager.test.js (hunk 3 above)
Comment (Stanislau-Snapok) [reply src/ModalManager.test.js]: Fixed

**Pull request merged**